test.js

```diff
@@ -5,3 +5,7 @@ console.log('Hello Universe');
 
 console.log('Hello other stuff');
 
+if(foo === "bar") {
+  console.log('stuff')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://release.k8s.dev.codacy.org/gh/franciscodua/codacy-test/pullRequest?prid=3017)
```suggestion
  console.log("stuff")
```